internal/winapi/ntfs.go

```diff
@@ -0,0 +1,61 @@
+package winapi
+
+//sys NtCreateFile(handle *uintptr, accessMask uint32, oa *ObjectAttributes, iosb *IOStatusBlock, allocationSize *uint64, fileAttributes uint32, shareAccess uint32, createDisposition uint32, createOptions uint32, eaBuffer *byte, eaLength uint32) (status uint32) = ntdll.NtCreateFile
```

Comment:
These calls work on more than just NTFS I'm pretty sure, FAT seems to be supported.